website/_kb/user-guide/users/profile-pictures.md

```diff
@@ -0,0 +1,16 @@
+---
+category: user-management
+parent_category: user-management
+title: Adding A Profile Picture
+slug: profile-pictures
+---
+
+If you login to Redash with a Google account, Redash will automatically fetch your Google Account profile picture. Alternatively, Redash uses [Gravatar](https://en.gravatar.com/) to handle profile pictures.
+
+Sign up for Gravatar with the email address you used to sign-in to Redash. When you're finished making your Gravatar, log out of Redash and then log back in. Your profile picture will automatically appear.
```

Comment:
There is no need to sign out, it will make no difference. The issue is the browser cache. Gravatars are cached for 5 minutes. Maybe it's enough to tell the user they should check back after 5 minutes to verify the image changed?